samples/drivers/lora_send/CMakeLists.txt

```diff
@@ -0,0 +1,7 @@
+cmake_minimum_required(VERSION 3.13.1)
```

Comment:
I missed this one in my first review. What about putting all the lora samples into `samples/drivers/lora` directory and then have `send` and `receive` subdirectories under it?

Reply:
Makes sense!